tests/serverpod_test_server/test_e2e_migrations/defaults/migrations_datetime_default_without_force_test.dart

```diff
@@ -102,17 +102,17 @@ void main() {
     });
 
     group(
-        'when removing the default value and making the column nullable without force,',
+        'when attempting to remove the default value and make the column nullable without force,',
```

Comment:
Suggestion: Same thing here this test should not require force as we are not risking failing the migration or losing any data.